index.js

```diff
@@ -62,9 +56,16 @@ GitHub.prototype = {
       cb = data;
       data = null;
     }
+
+    cb = typeof cb === 'function' ? cb : function noop () {};
```

Comment:
oops.. from @fake-user123, lol... just deleting the personal api token which was bugged because failing tests.